composer.json

```diff
@@ -32,7 +32,7 @@
         }
     },
     "require": {
-        "php": "^7.4 || ~8.0.0 || ~8.1.0",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0 || ~8.2.0",
```

Comment:
Hmm, I think the various `require-dev` will block you here. 

See https://github.com/laminas/laminas-view/blob/69ea122cd53f7839e58cb250975932332e542524/.laminas-ci.json#L7-L10 for an example of how to silence this temporarily.